examples/solids/README.rst

```diff
@@ -198,3 +198,43 @@ For example, consider a problem involving metals subject to gravity.
 
 One can choose units of displacement independently (e.g., :code:`-units_meter 100` to measure displacement in centimeters), but :math:`E` and :math:`\int \rho \bm g` have the same dependence on mass and time, so cannot both be made of order 1.
 This reflects the fact that both quantities are not equally significant for a given displacement size; the relative significance of gravity increases as the domain size grows.
+
+Diagnostic Quantities
+^^^^^^^^^^^^^^^^^^^^^
+
+Diagnostic quantities for viewing are provided when the command line options for visualization output, :code:`-view_soln` or :code:`-view_final_soln` are used.
+The diagnostic quantities include displacement in the :math:`x` direction, displacement in the :math:`y` direction, displacement in the :math:`z` direction, pressure, :math:`\operatorname{trace} \left( E \right)`, :math:`\operatorname{trace} \left( E^2 \right)`, :math:`\lvert J \rvert`, and strain energy density.
+The table below summarizes the formulations of each of these quantities for each problem type.
+
+.. list-table:: Diagnostic quantities
```

Comment:
I would specify:
```suggestion
.. list-table:: Diagnostic quantities for compressible or nearly-incompressible materials
```
Otherwise pressure would not have this formulation

Reply:
These are the diagnostic quantities for the three problem formulations that we currently have in the code. Our current formulations are compressible, but the plan is to eventually add incompressible to this table. I would then add a "incompressible hyperelasticity at finite strain" column to the table.

Reply:
Ok, but until then, I don't think it would hurt specifying that this table is not valid for incompressible. A reader who runs into this page may not know our internal development future plans

Reply:
If the user sets nu = 0.5, then the code NaNs and doesn't do anything, so they will figure it out pretty quick. But I'll add it somewhere that all of the derivations in the documentation are compressible.